playbook.md

```diff
@@ -1947,6 +1947,18 @@ incurring them.
 We manage expenses using Xero. For more information about how to do this, see
 the [guide on claiming expenses](/guides/claiming-expenses)
 
+## Reporting our time
+
+Everyone who works on a client project at dxw digital is responsible for reporting the time they spend on projects. We usually work with our clients on a [time and materials basis](https://www.gov.uk/guidance/how-to-pay-for-digital-outcomes-and-specialists-services#time-and-materials), so it’s important we can accurately track and report how our time is spent on client projects.
+
+We use [10,000ft](https://app.10000ft.com/) to schedule our work, which helps us to forecast our capacity over future weeks and months in a reliable way. Each member of our delivery teams uses the tool to record the days they work on client projects.
+
+We track our time at the same level of granularity that we bill it – for almost everyone in the team, that’s half days or whole days.
+
+If you’re not doing client work, then unless you’re working on one of a handful of named internal projects you don’t need to record your time against anything – the assumption is you’re doing valuable work, whether it’s learning, helping with recruitment or improving our tooling and processes, and we don’t need itemised time tracking to justify that.
+
+Our delivery leads lend a helping hand when delivery team members aren’t sure how to record their time – they’ve been doing this for a long while and have a well developed sense for it.
```

Comment:
Since this came up for me, is it worth adding something about how we track (or not) Fridays, sickness, holidays, and other working pattern related non-working? For instance, I wasn't aware that I wasn't supposed to be using "Out of office" for my regular days off.

Reply:
That's interesting, cause I thought we were supposed to be putting in our non-working days.

Reply:
I believe 10kft is now set up to understand part-time working schedules, so we only use "Out of office" for holiday. At least that's what I understood from talking to @evilstreak about it.

Reply:
@erbridge @johnwaterworth That's right - 10kft is set up to understand part-time working schedules. 

I'm going to write something about this, along with how we record sickness/annual leave, so this is clearer.

Reply:
I think 10000ft only understands some part time working patterns. For example, I work 4 days a week. My non-working day is normally Wednesday, but I do sometimes change day if needed. So I believe that it's useful for me to enter my non-working days to help with planning.

Reply:
In 10kft, if you're an admin, you can set per-week working patterns, so this perhaps this could be accomodated that way. For example mine has me manually set to 0 availability every other Monday. It's annoying that it's not possible to view what's set without modification access, though.

Reply:
I've been through 10,000ft and have added everyone's part time working patterns until December 2019, including those who work 3 days or 4 days per week and those who work different days on a fortnightly basis.

If someone changes their non working day, I think this should be down to the individual to record the day they actually worked on a client or named internal project, along with the day they weren't in the office as planned (as "Out of Office"), against the weekly reporting of their time - it's possible to record time on a day you shouldn't have been working on.